Frameworks/CoreFoundation/Stream.subproj/CFConcreteStreams.c

```diff
@@ -85,15 +125,13 @@ static void constructCFFD(_CFFileStreamContext *fileStream, Boolean forRead, str
 
 static Boolean constructFD(_CFFileStreamContext *fileStream, CFStreamError *error, Boolean forRead, struct _CFStream *stream) {
     int flags = forRead ? O_RDONLY : (O_CREAT | O_TRUNC | O_WRONLY);
+
+    char path[CFMaxPathSize];
 #if DEPLOYMENT_TARGET_WINDOWS
-    wchar_t path[CFMaxPathSize];
```

Comment:
it could but its not a change needed specifically for us as the normal impl of _NS_open calls _wopen. It was just to have better code reuse but will comment anyway.

---
In reply to: [83126093](https://github.com/Microsoft/WinObjC/pull/1137#discussion_r83126093) [](ancestors = 83126093)